modules/cart/src/Form/AddToCartForm.php

```diff
@@ -105,15 +105,17 @@ public function getBaseFormId() {
    * {@inheritdoc}
    */
   public function getFormId() {
-    $product_id = $this->entity->getPurchasedEntity()->getProductId();
+    static $form_id_suffix = 0;
```

Comment:
Wouldn't this make more sense as a static class property? Class methods generally don't use the static keyword.